### PR TITLE
Added User Card

### PR DESCRIPTION
**Pull request by RoyalSix**
#### This pull request addresses:
This PR adds all implementation for the user card section. #1946 

#### How to test this pull request:
Test the functionality listed in the issue above with the actual functionality. Also check against https://xd.adobe.com/view/105373ea-fd68-4cf5-8f07-61a3b67d4850/screen/96e44509-6925-40d6-8351-86e4162ba846/Local-Account-2
and #968 #966 #964

---
This change is [Reviewable](https://reviewable.io/reviews/unfoldingword-dev/translationcore/1953)

### CODE REVIEW

**Comment by mannycolon** [comment]
@RoyalSix waiting on tests to pass

**Comment by klappy** [comment]
*[src/js/components/home/usersManagement/LoginDoor43Account.js, line 18 at r1](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/1953#-KnZzhrgUUmj-kykSSbp:-KnZzhrgUUmj-kykSSbq:b-x311it) ([raw file](https://github.com/unfoldingword-dev/translationcore/blob/83e0e9c3963f7c70232aec8ccf2690fcb3abca27/src/js/components/home/usersManagement/LoginDoor43Account.js#L18)):*
> ```JavaScript
>         let title = <strong>Door43 Information</strong>
>         let text =
>             (<div style={{ padding: "0 20px" }}>
> ```

This formatting/whitespace should be updated, it's hard to follow in this layout.
```javascript
let text = (
  <div>
    <p>
      ...
      <br/>...
    </p>
    <p>
      ...
      <br/>...
    </p>
  </div>
);
```

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/1953)*

**Comment by klappy** [comment]
*[src/js/containers/home/LoginContainer.js, line 27 at r1](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/1953#-Kn_01WAVtgD3GiWw7Mm:-Kn_01WAVtgD3GiWw7Mn:b-yn708r) ([raw file](https://github.com/unfoldingword-dev/translationcore/blob/83e0e9c3963f7c70232aec8ccf2690fcb3abca27/src/js/containers/home/LoginContainer.js#L27)):*
> ```JavaScript
>             case 'local': return <CreateLocalAccount setView={this.setView} {...this.props} />;
>             case 'create': return <CreateDoor43Account setView={this.setView} {...this.props} />;
>             case 'login': return <LoginDoor43Account setView={this.setView} {...this.props} />;
> ```

Not sure why there aren't any errors here, but I thought switches required a default.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/1953)*

**Comment by klappy** [comment]
Reviewed 20 of 21 files at r1.
Review status: 20 of 21 files reviewed at latest revision, 2 unresolved discussions, some commit checks broke.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/1953)*

**Comment by RoyalSix** [comment]
Review status: 20 of 21 files reviewed at latest revision, 2 unresolved discussions, some commit checks broke.

---

*[src/js/components/home/usersManagement/LoginDoor43Account.js, line 18 at r1](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/1953#-KnZzhrgUUmj-kykSSbp:-Kn_4sj35BtfFuSVntXH:b-bw8e1h) ([raw file](https://github.com/unfoldingword-dev/translationcore/blob/83e0e9c3963f7c70232aec8ccf2690fcb3abca27/src/js/components/home/usersManagement/LoginDoor43Account.js#L18)):*
<details><summary><i>Previously, klappy (Christopher Klapp) wrote…</i></summary><blockquote>

This formatting/whitespace should be updated, it's hard to follow in this layout.
```javascript
let text = (
  <div>
    <p>
      ...
      <br/>...
    </p>
    <p>
      ...
      <br/>...
    </p>
  </div>
);
```
</blockquote></details>

Updated!

---

*[src/js/containers/home/LoginContainer.js, line 27 at r1](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/1953#-Kn_01WAVtgD3GiWw7Mm:-Kn_4mW_fA20WK-45bur:baeimo0) ([raw file](https://github.com/unfoldingword-dev/translationcore/blob/83e0e9c3963f7c70232aec8ccf2690fcb3abca27/src/js/containers/home/LoginContainer.js#L27)):*
<details><summary><i>Previously, klappy (Christopher Klapp) wrote…</i></summary><blockquote>

Not sure why there aren't any errors here, but I thought switches required a default.
</blockquote></details>

Haha true. I'm updating

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/1953)*

**Comment by mannycolon** [comment]
:lgtm:

---

Reviewed 6 of 6 files at r2.
Review status: all files reviewed at latest revision, 3 unresolved discussions, some commit checks broke.

---

*[src/js/components/login/LocalLogin.js, line 69 at r2](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/1953#-Kn_UERU7xIGtYYxQVQM:-Kn_UERU7xIGtYYxQVQN:b-afv72u) ([raw file](https://github.com/unfoldingword-dev/translationcore/blob/c5af380e9e9c59144f6d1d433d60e9d8df5bbf75/src/js/components/login/LocalLogin.js#L69)):*
> ```JavaScript
>                   style={{width: "0px"}}
>                   iconStyle={{fill: 'white'}}
>                   labelStyle={{color: "black", opacity: "0.7", fontWeight: "500"}}
> ```

do we wanna use the color scheme?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/1953#-:-Kn_XJbxk4C2Xh0VucKa:bnfp4nl)*
